MIGRATING.md

```diff
@@ -0,0 +1,153 @@
+
+Current effective required Node.js version is: ![node-current](https://img.shields.io/node/v/@splunk/otel?style=flat-square)
+
+## Migration steps
+
+### Instrumented libraries
+
+With the exception of [explicitly listed limitations](#known-limitations) we aim to support all libraries supported by
+signalfx-nodejs-tracing. To find an equivalent auto-instrumentation open <https://opentelemetry.io/registry/> and for
+each instrumentation
+[from `signalfx-nodejs-tracing`'s README](https://github.com/signalfx/signalfx-nodejs-tracing/#requirements-and-supported-software)
+search by the name of the library in the registry.
+
+For example, if you'd like to migrate instrumentation for `mysql`, go to
+[https://opentelemetry.io/registry/?s=**mysql**&component=&language=**js**#](https://opentelemetry.io/registry/?s=mysql&component=&language=js#).
+
+Once you have identified an instrumentation package, install it using `npm` or `yarn`.
+
+- If the package is [on this list](./README.md#default-instrumentation-packages-), it
+  will be enabled automatically.
+- if it's not on the list, follow the steps for
+  [installing other instrumentation packages](./README.md#custom-instrumentation-packages).
+
+### Environment variables
+
+Rename environment variables:
+
+| Old environment variable           | New environment variable             | notes |
+| ---------------------------------- | ------------------------------------ | ----- |
+| SIGNALFX_ACCESS_TOKEN              | SPLUNK_ACCESS_TOKEN                  |       |
+| SIGNALFX_SERVICE_NAME              | OTEL_SERVICE_NAME                    |       |
+| SIGNALFX_ENDPOINT_URL              | OTEL_EXPORTER_JAEGER_ENDPOINT        | if Jaeger is used |
+| SIGNALFX_ENDPOINT_URL              | n/a                                  | OTLP is not implemented yet |
+| SIGNALFX_RECORDED_VALUE_MAX_LENGTH | SPLUNK_MAX_ATTR_LENGTH               |       |
+| SIGNALFX_TRACING_DEBUG             | no direct equivalent                 | see [instrumentation logs](#instrumentation-logs) |
+| SIGNALFX_SPAN_TAGS                 | OTEL_RESOURCE_ATTRIBUTES             | format needs to be changed to `key1=val1,key2=val2` |
+| SIGNALFX_LOGS_INJECTION            | SPLUNK_LOGS_INJECTION                | |
+| SIGNALFX_LOGS_INJECTION_TAGS       | OTEL_RESOURCE_ATTRIBUTES             | there's no direct equivalent, but values specified in `OTEL_RESOURCE_ATTRIBUTES` will also be used for logs injection |
+| SIGNALFX_ENABLED_PLUGINS           | n/a                                  | see [the README section about instrumentations](./README.md#custom-instrumentation-packages) |
+| SIGNALFX_SERVER_TIMING_CONTEXT     | SPLUNK_TRACE_RESPONSE_HEADER_ENABLED | |
+| SIGNALFX_TRACING_ENABLED           | OTEL_TRACE_ENABLED                   | |
+
+### Programmatic configuration
+
+Update these programmatic configuration options:
+
+| Old property             | New property            | Notes |
```

Comment:
Instead of "old" and "new" I'd use the names "OpenTracing" and "OpenTelemetry" or `signalfx-tracing` and `@splunk/otel`.